src/main/java/space/obminyashka/items_exchange/service/impl/SendGridService.java

```diff
@@ -35,6 +38,11 @@ public void sendMail(String emailTo, EmailType subject, Locale locale) throws IO
         log.debug("[SendGridService] A sent email result. STATUS: {} BODY: {}", response.getStatusCode(), response.getBody());
     }
 
+    @Override
+    public void validateEmail(UUID validationCode) throws EmailTokenNotExistsException, EmailTokenExpiredException {
+        //TODO
```

Comment:
You shouldn't create an empty TODO for any cases. It needs to have an explanation and the task number that covers that todo